language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/completion/latest/ClassDefContextTest.java

```diff
@@ -41,6 +41,6 @@ public String getTestResourceDir() {
 
     @Override
     public List<String> skipList() {
-        return Arrays.asList("config3.json", "config8.json");
+        return Arrays.asList("config3.json", "config8.json", "config12.json", "config13.json");
```

Comment:
Can we create an issue to track this test disabling?

Reply:
Issue created https://github.com/ballerina-platform/ballerina-lang/issues/26017